### PR TITLE
using 3rd party services to send wallpaper to Travis(meilix)

### LINKED ISSUE #398: Replacing the way to send the wallpaper, logo, and zipfiles from the webapp to build server
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Expected Behaviour**
In the current build, wallpaper in stored in the wallpapers folder in the server and previously this was accessed during the build by accessing `https://meilix-generator.herokuapp.com/`. This can cause same wallpaper for two different users which have build the ISO using the generator. 

**Would you like to work on the issue?**
Open

### PR DESCRIPTION
**Pull request by alok760**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
sending wallpaper files to Travis 

#### Changes proposed in this pull request:

- Send wallpaper using transfer.sh
- If transfer.sh fails, send using 0x0.st
- if both fails, upload files to host server(subject race around)

Fixes #398

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -59,9 +60,26 @@ def upload_wallpaper(wallpaper):
...
+                    print("upload failed(transfer.sh) \n retrying(0x0.st)")
+                    wallpaper.seek(0)
+                    response = requests.post('https://0x0.st', files= {'file': (filename, wallpaper),})
+                    url = response.text
+                    print(url)
+                except:
+                    # Saving wallpaper to host
+                    wallpaper.seek(0)
+                    wallpaper.save(os.path.join(app.config['UPLOAD_FOLDER'] + app.config['WALLPAPER_FOLDER'], filename))
+                    os.rename(UPLOAD_FOLDER + WALLPAPER_FOLDER + filename, UPLOAD_FOLDER + WALLPAPER_FOLDER + 'wallpaper')
```
Comment (xeon-zolt) [inline app.py]: if we are sending url then if end up on last except we don't have a url

File: app.py [hunk 2]
```diff
@@ -130,7 +148,7 @@ def index():
 def output():
     if flag:
         if os.environ['TRAVIS_TAG']:  # if TRAVIS_TAG have value it will proceed
-            trigger_code = build.send_trigger_request(os.environ['email'], os.environ['TRAVIS_TAG'], os.environ['event_url'],os.environ['TRAVIS_SCRIPT'], os.environ['recipe'], os.environ['processor'], os.environ['feature'])
+            trigger_code = build.send_trigger_request(os.environ['email'], os.environ['TRAVIS_TAG'], os.environ['event_url'],os.environ['TRAVIS_SCRIPT'], os.environ['recipe'], os.environ['processor'], os.environ['feature'], os.environ['wallpaper_url'])
```
Comment (xeon-zolt) [inline app.py]: now we need to add wget to meilix and use 'o' flag to save to same name for file which is to be used by theme

Comment (xeon-zolt) [reply app.py]: this is just a reminder

File: app.py (hunk 1 above)
Comment (xeon-zolt) [reply app.py]: this is to be fixed

**Comment by pranav1698** [comment]
@alok760 please add a deployment link to test the changes made because it can lead to unexpected errors in the main generator,
After the merge,
we are getting this pr:
Screenshot from 2019-03-16 10-13-18
I am opening an issue for the same

**Pull request merged**